src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -523,6 +534,9 @@ ov::AnyMap get_default_generate_config(const std::shared_ptr<ov::Model>& model,
     if (npudesc.has_value() && npudesc->arch == "4000") {
         config.emplace("NPU_DPU_GROUPS", 4);
     }
+    if (hint == GenerateHint::FAST_COMPILE) {
```

Comment:
Let's check if it doesn't break anything on arch<4000

Reply:
Looks there is no regressions on MTL system. While performance(tokens rate) of LLama2 increased at least by  10%